synthtool/gcp/common.py

```diff
@@ -38,8 +41,8 @@ def __init__(self):
 
     def _generic_library(self, directory: str, **kwargs) -> Path:
         # load common repo meta information (metadata that's not language specific).
+
         if "metadata" in kwargs:
```

Comment:
Why do we search for "metadata" in kwargs?
Why not declare an argument metadata=None?